string soma
string soma_config
string region_id
time start_time
time end_time
int32 count
string detection_type
int32 max_value
